src/Chapter3/Pagination.hs

```diff
@@ -0,0 +1,22 @@
+module Chapter3.Pagination where
+import Data.Char
+import Data.List
+
+data Pages = MkPage {
+    prev :: String,
+    current :: String,
+    next :: String
+}deriving Show
+
+displayPagination :: Int -> Int -> Int -> Int -> Pages
```

Comment:
Are the semantics of this function really clear?

Reply:
And what is one supposed to do with the `Pages` data structure? Is this really solving the original problem?